bitpacker/src/bitpacker.rs

```diff
@@ -48,7 +48,7 @@ impl BitPacker {
 
     pub fn flush<TWrite: io::Write + ?Sized>(&mut self, output: &mut TWrite) -> io::Result<()> {
         if self.mini_buffer_written > 0 {
-            let num_bytes = (self.mini_buffer_written + 7) / 8;
+            let num_bytes = self.mini_buffer_written.div_ceil(8);
```

Comment:
Here it does not matter, but this has a small micro impact on performance.

Reply:
ah good to know, I thought it would lead to the same instruction